numba/unicode.py

```diff
@@ -1286,6 +1288,85 @@ def impl(a):
     return impl
 
 
+@register_jitable
+def _handle_capital_sigma(data, length, idx):
+    """
+    https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Objects/unicodeobject.c#L9856-L9883
+    """
+    c = 0
+    j = idx - 1
```

Comment:
Not sure what these two lines are for?

Reply:
`c` can be required for the case when the first loop is skipped due to `j`.
I will make the function as it implemented for `str.lower()` (https://github.com/numba/numba/pull/4651).